superset/dataframe.py

```diff
@@ -134,8 +134,12 @@ def datetime_conversion_rate(cls, data_series):
 
     @classmethod
     def is_date(cls, dtype):
-        if dtype.name:
-            return dtype.name.startswith('datetime')
+        if dtype and dtype.name:
+            return any([
+                dtype.name.startswith(s.lower())
+                for s in ['date', 'time']
```

Comment:
Should we do `dtype.name.startswith(s.lower().lstrip('_'))` instead, to catch `__time` as a timestamp as well?

Reply:
oops, realized I was lowering the hard coded strings...